### PR TITLE
Pin camouflage to v0.9 and ensure pytest-benchmark>=4

### PR DESCRIPTION
**Pull request by dagardner-nv**
v0.10 was released on 2022-10-25 and completely breaks our tests which depends on it.
pytest-benchmark 4.0 fixes an incompatibility with pytest 7.2

### CODE REVIEW

File: docker/conda/environments/requirements.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 # Packages listed here should also be listed in setup.py
 git+https://github.com/nv-morpheus/dfencoder.git@branch-22.09#egg=dfencoder
 nvidia-pyindex
+pytest-benchmark>=4.0
```
Comment (mdemoret-nv) [inline docker/conda/environments/requirements.txt:9]: Why is this here instead of in the conda environment?

**Review by mdemoret-nv — COMMENTED** [review]
Quick question

File: docker/conda/environments/requirements.txt (hunk 1 above)
Comment (dagardner-nv) [reply docker/conda/environments/requirements.txt:9]: I originally added it in the yaml, but conda would report that nothing provides the package. Putting it in the requirements worked.

Comment (dagardner-nv) [reply docker/conda/environments/requirements.txt:9]: At the time yesterday pytest-benchmark 4.0 hadn't been uploaded to conda-forge, it is now, so I could move it back to the yaml.

**Review by mdemoret-nv — APPROVED** [review]
LGTM

**Comment by dagardner-nv** [comment]
@gpucibot merge
